designsystem/src/commonMain/kotlin/org/hisp/dhis/mobileui/designsystem/theme/Theme.kt

```diff
@@ -41,3 +44,15 @@ fun DHIS2Theme(
         content = content
     )
 }
+
+object CustomDHISRippleTheme : RippleTheme {
```

Comment:
When Icon Buttons Issue is Merged we can rebase this branch and access it through Ripple class